content/en/user-guide/aws/pipes/index.md

```diff
@@ -0,0 +1,122 @@
+---
+title: "EventBridge Pipes"
+linkTitle: "EventBridge Pipes"
+description: Get started with EventBridge Pipes on LocalStack
+---
+
```

Comment:
@joe4dev  This should probably include a fairly big ALPHA/EXPERIMENTAL banner here since we can't provide support for this yet.